Week2/homework/map-filter.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  numbers = numbers.filter(x => x % 2).map(x => x * 2);
```

Comment:
Here, we are just working with numbers that have no higher meaning than being numbers. So the use of the algebraic name `x` is entirely appropriate here.